app/helpers/quadicon_helper.rb

```diff
@@ -29,12 +29,24 @@ module QuadiconHelper
     'template'                  => {:fonticon => 'pficon pficon-template', :background => '#336699'},
   }.freeze
 
-  def self.status_img(status)
+  def self.provider_status(status, enabled = true)
+    # If the provider is suspended, we don't care about the status itself
+    unless enabled
+      return {
+        :fonticon => 'pficon pficon-asleep',
+        :tooltip  => _('This provider is paused, no data is currently collected from it')
```

Comment:
Based on what is described in the [issue](https://github.com/ManageIQ/manageiq/issues/17489) @slemrmartin created, I feel like the tooltip message is incorrect:

> What does it mean - suspend provider? What features should be disabled?
> 
>  *  Ems Refresh
>  *  SmartState analysis of: Vm, Template, Host, ..(?)
>  *  Metrics Collection (C&U)

It's not the whole provider that is suspended, but the data collection.
Correct me if I'm wrong, @slemrmartin.

Reply:
Yes, you're right, but the second part of tooltip describes what it means, maybe there can be replaced ',' for '-' or '->' for make it more obvious.
Important is, unify the words "paused" / "suspended" according to button names in toolbars

Reply:
@slemrmartin @skateman I'd suggest something like _"Data collection for this provider is paused"_.

/cc @epwinchell

Reply:
@romanblanco That sounds good.